src/compressed_tensors/compressors/model_compressors/model_compressor.py

```diff
@@ -421,6 +421,13 @@ def compress_model(self, model: Module):
 
                 module.quantization_status = QuantizationStatus.COMPRESSED
 
+        # TODO: consider sparse compression to also be compression
```

Comment:
are there any other places in the code that don't make this assumption? What is the TODO here?

Reply:
Ideally, a model which only uses sparsity compression is still considered "compressed". The TODO is to figure out other places in the code that make this assumption

Reply:
ok, as a nit, I assume a TODO means to modify code in or around this segment, or attached to an issue #, otherwise it might not get pruned if/when those changes are made. but nbd, not sure if we have a convention or rule of thumb for this

Reply:
I agree that my use of todos is fairly liberal. This todo does mean "figure out how we can change this particular code to change compression status, even in sparsity-only cases"